src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1189,6 +1189,116 @@ groovyscript.wiki.naturesaura.spawning.entity.value=Sets the entity spawned
 groovyscript.wiki.naturesaura.spawning.removeByEntity=Removes all Altar of Birthing recipes that summon the given entity
 groovyscript.wiki.naturesaura.spawning.removeByName=Removes the Altar of Birthing recipe with the given name
 
+# ProdigyTech
+groovyscript.wiki.prodigytech.atomic_reshaper.title=Atomic Reshaper
+groovyscript.wiki.prodigytech.atomic_reshaper.description=Uses hot air and Primordium to convert items, can have a weighted random based output
```

Comment:
could you put a period at the end of the `description` lang keys? its not included by default, and the sentence should have one. honestly, should probably automatically insert a period at the end, its weird and the only lang key that this applies to.